webapp/package.json

```diff
@@ -76,6 +76,7 @@
   },
   "dependencies": {
     "@calls/common": "github:mattermost/calls-common#25619914bcfe6347101b29204b2976518742e81e",
+    "@mattermost/compass-icons": "0.1.40",
```

Comment:
I worked so hard on removing this dependency back then, do we really need to add it back? :p

Reply:
Nope, don't need it at all -- it doesn't seem to work anyway (we only get the one that's included in server), so it's mostly useless to us, considering we add new icons on almost every UI update. :)